lib/pymedphys/_mosaiq/connect.py

```diff
@@ -151,7 +151,11 @@ def delete_credentials(sql_server_and_port):
 
 
 def _connect_with_credential_then_prompt_if_fail(
-    sql_server_and_port, user_input=input, password_input=getpass, output=print
+    sql_server_and_port,
+    user_input=input,
+    password_input=getpass,
+    output=print,
+    database="MOSAIQ",
```

Comment:
The keyring password "key" will need to be adjusted to have the database name added to it so that login credentials for one database on the same hostname:port combo are not attempted to be used for a different database name that happens to be served on that same combo.

Reply:
you mean instead of 

`keyring.get_password("MosaiqSQL_username", storage_name)`

change to

`keyring.get_password(f"{database_name}SQL_username", storage_name)`

Reply:
Either there, or within the storage name. Figured I'd leave the design decision on that front to your discretion 🙂.

Reply:
sorry I altered the approach to put it in the storage_name, though I'm not sure I understand how the storage name shows up in credential manager

Reply:
> sorry I altered the approach to put it in the storage_name, though I'm not sure I understand how the storage name shows up in credential manager

It's all good. I am happy to go with this approach. If we find out down the line we'd like to adjust it further we are certainly free to do that.

Reply:
I made a change to the order of parameters for keyring.get_password/set_password  -- please have a look and tell me if it makes sense to you.  I think the entries in Windows Credential manager make more sense if "storage_name" (first parameter) is the one with the server/database, and "user_name" (second parameter) is _PyMedPhys_MosaiqSQL_xxx_ -- what do you think?

Reply:
Yup, that certainly makes sense to me. I'll checkout your PR now and make sure things still work happily :)